cmd/snapctl/plugin.go

```diff
@@ -169,11 +169,19 @@ func listPlugins(ctx *cli.Context) error {
 	}
 	w := tabwriter.NewWriter(os.Stdout, 0, 8, 1, '\t', 0)
 	if ctx.Bool("running") {
+		if len(plugins.AvailablePlugins) == 0 {
+			fmt.Println("No running plugins found.")
```

Comment:
To be consistent with your other messages shouldn't you also add `Have you started a task?`? 😄